src/serac/physics/contact/contact_data.hpp

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) 2019-2023, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+/**
+ * @file contact_data.hpp
+ *
+ * @brief Class for storing contact data
+ */
+
+#pragma once
+
+#include <mfem/fem/pgridfunc.hpp>
+#include "mfem.hpp"
+
+#include "serac/serac_config.hpp"
+#include "serac/physics/contact/contact_config.hpp"
+#include "serac/physics/state/finite_element_dual.hpp"
+#include "serac/physics/state/finite_element_state.hpp"
+#ifdef SERAC_USE_TRIBOL
+#include "serac/physics/contact/contact_pair.hpp"
+#endif
+
+namespace serac {
+
+namespace contact {
+
+/**
+ * @brief Default contact options: frictionless mortar with penalty = 1000
+ * enforcement
+ */
+const ContactOptions default_contact_options = {.method      = ContactMethod::SingleMortar,
+                                                .enforcement = ContactEnforcement::Penalty,
+                                                .type        = ContactType::Frictionless,
+                                                .penalty     = 1.0e3};
+
+}  // namespace contact
+
+/**
+ * @brief This class stores contact pair data and interacts with Tribol
+ */
+class ContactData {
+public:
```

Comment:
if the purpose of this class is to interact with Tribol, should we just name it "TribolInterface" or similar? `ContactData`, `ContactPairs` and `ContactConfig` all sound like the same thing.